debian/rules

```diff
@@ -1,105 +1,40 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# This file was originally written by Joey Hess and Craig Small.
-# As a special exception, when this file is copied by dh-make into a
-# dh-make output file, you may use that output file without restriction.
-# This special exception was added by Craig Small in version 0.37 of dh-make.
 
-# Uncomment this to turn on verbose mode.
-#export DH_VERBOSE=1
+BUILDDIR = build
+DESTDIR = debian/nest
+DISTRIBUTION = $(shell lsb_release -sc)
+PACKAGEVERSION = nest-master
+INSTALLDIR = \/$(BUILDDIR)\/$(PACKAGEVERSION)\/$(DESTDIR)
+
+CMAKE_FLAGS = -DCMAKE_INSTALL_PREFIX:PATH=$(DESTDIR)/usr -Dwith-python=3 -DCMAKE_INSTALL_LIBDIR:PATH=lib
+CMAKE_FLAGS += -DCMAKE_CXX_COMPILER=g++ -DCMAKE_C_COMPILER=gcc
+CMAKE_FLAGS += -Dwith-mpi:BOOL=On
+CMAKE_FLAGS += -Dwith-openmp:BOOL=On
+CMAKE_FLAGS += -Dwith-gsl:BOOL=On /usr/local/lib
+CMAKE_FLAGS += -Dwith-libneurosim:BOOL=On /usr/local/lib
+CMAKE_FLAGS += -Dwith-music=On /usr/lib
+
+%:
+	LC_ALL=en_US.UTF-8 dh ${@} --with python3 --builddirectory=$(BUILDDIR)
+	#dh $@
+
+override_dh_auto_clean:
+	dh_clean
+	rm -rf $BUILDDIR
+override_dh_auto_test:
+override_dh_installdirs:
+override_dh_installdocs:
+override_dh_installinfo:
+override_dh_shlibdeps:
+override_dh_auto_configure:
+override_dh_auto_build:
+override_dh_auto_install:
+    # This is a little bit tricky
+    # @todo Check if this really is problem or only in building the packages
+	sed -i 's/site-packages/dist-packages/g' ./cmake/ProcessOptions.cmake
+	sed -i 's/NEST_INSTALL_PREFIX \"\/\"/\"\/usr\/"/g' ./sli/slistartup.cc
+	cmake $(CMAKE_FLAGS) .
+	make
+	make install
+	rm -f $(DESTDIR)/usr/bin/nest_serial $(DESTDIR)/usr/bin/nest_indirect $(DESTDIR)/usr/bin/sli $(DESTDIR)/usr/bin/nest_vars.sh
```

Comment:
Why are you deleting these? At least `sli` is useful to get the helpdesk shown if you compile with MPI.

Reply:
You're totally right. This line can go away.
Instead I have inserted a line in which `nest_wars.sh` is adjusted and the correct paths are set. Then `nest_vars.sh` is sourced in `etc/profile.d/nest-simulator.sh` as you suggested.

Since NEST is installed in the default paths on the system, it will work after installation. 
If you have problems viewing the documentation, the paths must be set explicitly. There are now two simple ways to do this:

1. log off and on again, or
2. source `nest_vars.sh`  with `. nest_ vars.sh`